changelog/db.changelog.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>   
+<databaseChangeLog  
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"  
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"  
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">  
+  <include  file="changelog/db.changelog-0.0.xml"/>
+  <include  file="changelog/db.changelog-1.0.xml"/>
```

Comment:
Every file must be referenced in this wrapper file or it will be ignored by Liquibase.